content/source/docs/enterprise/api/run.html.md

```diff
@@ -39,13 +41,15 @@ Key path                    | Type   | Default | Description
 Status  | Response                               | Reason
 --------|----------------------------------------|-------
 [200][] | [JSON API document][] (`type: "runs"`) | Successfully created a run
+[401][] | [JSON API error object][]              | Tried to create run with organization token (use a user or team token instead)
```

Comment:
It also might be that the token isn't set. I think I'd note "Authentication failure; token not set or wrong token type (use a user or team token)". This would apply throughout.